userspace/libscap/scap.h

```diff
@@ -18,6 +18,9 @@ limitations under the License.
 */
 
 #pragma once
+#ifdef MUSL_OPTIMIZED
+#include <bits/alltypes.h>
```

Comment:
This should be `<time.h>` and it should probably be included at the point where the types are needed rather than here.